File: src/main/java/ru/openbank/meetup/demo/java/slide7/RunUseCase.java
package ru.openbank.meetup.demo.java.slide7;

import com.fasterxml.jackson.databind.JsonNode;

public class RunUseCase {
    private static boolean isValidNode(JsonNode jsonNode) {
        return jsonNode != null && (jsonNode.isObject() || jsonNode.isArray());
    }
}

File: src/main/kotlin/ru/openbank/meetup/demo/kotlin/slide7/RunUseCase.kt
package ru.openbank.meetup.demo.kotlin.slide7

import com.fasterxml.jackson.databind.JsonNode

//private fun isValidNode(jsonNode: JsonNode?) = jsonNode?.also { it.isObject || it.isArray } ?: false

private fun isValidNode(jsonNode: JsonNode?) = jsonNode?.let { it.isObject || it.isArray } ?: false

